hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestHoodieSqlBase.scala

```diff
@@ -92,6 +96,19 @@ class TestHoodieSqlBase extends FunSuite with BeforeAndAfterAll {
     assertResult(true)(hasException)
   }
 
+  protected def checkExceptionContain(sql: String)(errorMsg: String): Unit = {
+    var hasException = false
+    try {
+      spark.sql(sql)
+    } catch {
+      case e: Throwable =>
+        assertResult(true)(e.getMessage.contains(errorMsg))
+        hasException = true
+    }
+    assertResult(true)(hasException)
```

Comment:
are we able to use `assertThrows()` here from junit 5? or leave it the scala way also fine.

Reply:
Just leave it for now. Some UTs don't have the `expectedType` which is the first parameter in `org.junit.jupiter.api.Assertions.assertThrows()`, we just wanna to compare the exception message.